src/kiwixapp.cpp

```diff
@@ -111,6 +111,16 @@ void KiwixApp::init()
             m_library.asyncUpdateFromDir(dir);
         }
     }
+    
+    QStringList tabs = m_settingsManager.getTabList();
+    m_settingsManager.setTabList(QStringList{});
+    if (m_settingsManager.getReopenTab())
+    {
+      /* Restart a new session to prevent duplicate records in openURL */
+      for (const auto &zimFile : tabs)
+        if (!zimFile.isEmpty())
+          openUrl(zimFile);
+    }
```

Comment:
Maybe we want to check the saved tabs (url) are still valid before open them.

If user delete a zim file after quitting the application, we will try to open a zim file which is not present anymore.
`openUrl` will open a new tab and it will be a 404.
It may be what we want or we may not open a tab at all in this case...

Reply:
On my side, the application simply segfaults if the user deletes the zim file. Tested on a fresh main as well. This might be a new issue?

Reply:
Actually, I will fix them right here, as using openBook to handle an error like this is going to be the way to go anyways.

Reply:
The crash is probably because of https://github.com/kiwix/kiwix-desktop/issues/893

Reply:
Got it. I will remove that nullptr check then.

Reply:
Done